packages/algob/test/builtin-tasks/deploy.ts

```diff
@@ -36,14 +37,14 @@ scripts directory: script 2 executed
       noCompile: true
     });
     const snapshot1 = loadCheckpoint("./scripts/1.js");
-    assert.deepEqual(snapshot1.default.metadata, {
+    assert.deepEqual(snapshot1.default.metadata, toMap({
       "script 1 key": "script 1 value"
-    });
+    }));
```

Comment:
the way how to create Map directly is:
```
let first = new Map([
  [1, 'one'],
  [2, 'two'],
  [3, 'three'],
])
```

Reply:
Let's use a direct syntax.

Reply:
It is not also a direct syntax.
It creates a map from a list of lists which probably is more objects than simply using one object and a function.

Reply:
Also this:
![Screenshot at 2020-08-30 21-35-32](https://user-images.githubusercontent.com/1641263/91666847-f58ac700-eb08-11ea-89bd-e95fc95c3808.png)